app/services/gcp/kms.py

```
@@ -0,0 +1,197 @@
+        logger.warning("GCP KMS client not available, cannot decrypt")
+        return None
+
+    try:
+        key_name = _get_key_name()
+        ciphertext = base64.b64decode(ciphertext_b64)
+
+        response = client.decrypt(request={"name": key_name, "ciphertext": ciphertext})
+
+        plaintext = response.plaintext.decode("utf-8")
+        logger.debug("GCP KMS decryption successful")
+        return plaintext
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)